steam_buddy/platforms/flathub.py

```diff
@@ -108,9 +108,9 @@ def get_shortcut(self, content):
             'name': content.name,
             'hidden': False,
             'banner': self.get_image_file_path(content.content_id),
-            'params': content.content_id,
-            'cmd': "flatpak run",
-            'tags': ["Flathub"],
+            'cmd': "flatpak run " + content.content_id,
```

Comment:
By making this change, the Steam app id will also change now, and also possibly in the future if the Flathub id of the program changes (this has already happened once before).

I have plans to change the command in the future so I don't want to change the app id too much (means loss of tags etc.)

Reply:
Steam ID will change but only for newly installed or updated games from Flathub, existing ones won't be affected since the shortcut is already created on the yaml file. The same goes for id changes on Flathub packages, that one we cannot control and our solution won't work either since we may need to change the id on the image file that whitelists it inside steam-buddy.

Since the appid won't change on already installed apps, the tags and such won't be affected. But maybe the solution is to wrap the `cmd` line per app in a predictable way.

Reply:
Ah, you are correct that it probably wouldn't cause problems. I still prefer to keep the content id in the launch options to be consistent with all the other shortcuts.

Reply:
Mhhhh. Maybe we should change all to this pattern? The `params` entry should be used for other kind of stuff IMHO.

Reply:
You can still add your own parameters though. This doesn't prevent it.

And no I do not plan to change this. For ROMs it is very possible that you upload a new ROM to replace your current one and the file name could potentially change. I do not want a file name change to result in a different steam app id.

Reply:
I may re-evaluate this idea later when we update the steam-buddy CLI and re-work how games are being launched.

Reply:
Fair enough. Will revert this change.
The StartDir one needs to be added, so I'll keep this PR open.